misc/scripts/upgrade_test.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+    source ../misc/scripts/command.sh
+    MANAGED_PLUGIN_NAME="vsphere:latest"
+    E2E_Tests="github.com/vmware/docker-volume-vsphere/tests/e2e"
+    GO="go"
+
+    get_vib_url() {
+        echo "Get version $1"
+        if [ $1 == "0.14" ]
```

Comment:
how about the second approach it mentioned at https://github.com/vmware/docker-volume-vsphere/pull/1569#discussion_r126988686?